src/index.css

```diff
@@ -116,6 +181,36 @@ h1 {
     margin-left: 24px;
     margin-right: 24px;
   }
+
+  nav ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: start;
+    align-items: center;
+    height: 74px;
+    margin-left: 24px;
+    padding-bottom: 0;
+    padding-top: 0;
+  }
+
+  nav li {
+    display: inline;
```

Comment:
The list items are in a flex container. So, I don't think it's necessary to have `display: inline`